packages/rpc-core/src/rpc-methods/getMinimumBalanceForRentExemption.ts

```diff
@@ -0,0 +1,15 @@
+import { Commitment, LamportsUnsafeBeyond2Pow53Minus1, RpcResponse, U64UnsafeBeyond2Pow53Minus1 } from './common';
+
+type GetMinimumBalanceForRentExemptionApiResponse = RpcResponse<LamportsUnsafeBeyond2Pow53Minus1>;
+
+export interface GetMinimumBalanceForRentExemptionApi {
+    /**
+     * Returns the minimum balance to exempt an account of a certain size from rent
+     */
+    getMinimumBalanceForRentExemption(
+        size: U64UnsafeBeyond2Pow53Minus1,
```

Comment:
note this type is technically uszie but i dunno if we care about that distinction in js, in practice it should always be equivalent to u64. also note the rpc docs indicate its optional but that seems silly

Reply:
Actually it will throw you an error if you provide no args, so we should prob throw a flare up on the JSON RPC docs. 

This is one of 3 mistakes we've found in the docs (that I know about). I think I'll just collect this info and make a PR

Reply:
We've been using the U64/bigint types for usize so this is right. And yea agreed looks like an error in the RPC docs